src/components/DFR3Viewer.jsx

```diff
@@ -759,522 +505,286 @@ class DFR3Viewer extends React.Component {
 			});
 		}
 		return params;
-	}
+	};
+
+	const handleCloseMetadata = () => {
+		setMetadataClosed(true);
+	};
 
-	closeMetadata() {
-		this.setState({
-			metadataClosed: true
+
+	// curve list
+	const curveList = [...dfr3Curves];
+	let curvesWithInfo = [];
+	if (curveList.length > 0) {
+		curveList.map((DFR3Curve) => {
+			if (is3dCurve(DFR3Curve)){
+				console.log(DFR3Curve.id)
+			}
+			curvesWithInfo.push(DFR3Curve);
```

Comment:
Since you are checking on the fly, don't you want to remove that part?

Reply:
Ahh, yeah, I forgot lol I wanted to find out what curve was with 3d plot... Ill remove it. Thanks